### PR TITLE
feat: added avatar and badge components

### PR DESCRIPTION
**Pull request by srishti-maurya**
These are just components, the documentation is not added yet! 

Screenshots is attached for reference below.

Any suggestion/feedback would be highly appreciated.

View Live at : https://srishti-maurya.github.io/Plan-it-Component-Library/

screencapture-127-0-0-1-5500-components-badge-html-2022-01-29-15_21_30
screencapture-127-0-0-1-5500-components-avatar-html-2022-01-29-15_21_09

### CODE REVIEW

**Review by pritam-kr — APPROVED** [review]
looks clean!  so you're doing separately I mean to say first build a component on another page.

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.avatar-md {
+  height: 80px;
+  width: 80px;
+}
+
+.avatar-lg {
+  height: 120px;
+  width: 120px;
+}
+
```
Comment (vijaytembugade) [inline styles/styles.css:24]: You can give heights and width with rems. Just saying ,as it might be helpful in future.

**Comment by vijaytembugade** [comment]
Other things looks perfect. Badges are amazing.

File: styles/styles.css (hunk 1 above)
Comment (srishti-maurya) [reply styles/styles.css:24]: Giving images the sizes in pixels I thought would be better. I'll look into it again once I'll be building image component.

**Pull request merged**